src/api/firebase.js

```diff
@@ -91,3 +91,19 @@ export async function createNewList(listId) {
 	}
 	return response;
 }
+
+export async function checkIfListExists(listId) {
+	const listCollectionRef = collection(db, listId);
+
+	try {
+		const querySnapshot = await getDocs(listCollectionRef);
+
+		// If the collection exists and has documents, return the snapshot
+		if (!querySnapshot.empty) {
+			return querySnapshot;
+		}
```

Comment:
If you needed the `null`, then it's ok to leave it!

I was suggesting to remove the `else` statement, but not what was inside of it 😅 

```js
if (!querySnapshot.empty) {
  return querySnapshot
}

// you don't need to write the ELSE because you returned early inside the if(...). I hope this makes sense, let me know if it doesn't!
return null
```

Reply:
😅 💯